prac_05/emails.py

```diff
@@ -1,5 +1,32 @@
 """
 CP1404/CP5632 Practical
-Emails
+Emails in dictionaries
 """
 
+
+def main():
+    """ Get email and name from user and store in a dictionary"""
+    email_dict = {}
+
+    email = input("Email: ")
+    while email:
+        name = get_name(email)
+        name_correct_input = input(f"Is your name {name}? (Y/n) ").lower()
+        if name_correct_input and name_correct_input != "y":
```

Comment:
Forgive me if I'm wrong but it seems quite pointless to have name_correct_input twice, the first one isn't really doing anything. Perhaps if name_correct_input != Y or name_correct_input.upper != "Y"